lua/fff/main.lua

```diff
@@ -141,6 +141,7 @@ function M.setup_global_autocmds()
             if stat and stat.type == 'file' then
               local relative_path = vim.fn.fnamemodify(file_path, ':.')
               pcall(fuzzy.access_file, relative_path)
+              pcall(fuzzy.refresh_git_status)
```

Comment:
Refresh git status is a costly operation counting all the files, why do we need to call it on every file open?